server/applicationPattern/applicationPattern/rest/client/RequestBuilder.js

```diff
@@ -33,22 +34,13 @@ exports.BuildAndTriggerRestRequest = async function (operationClientUuid, method
             "body :" + JSON.stringify(requestBody) + "response code:" + response.status)
         return response;
     } catch (error) {
-        console.log(error);
         if (error.response) {
-            if (error.response.status) {
```

Comment:
This code we introduced to solve this problem https://github.com/openBackhaul/ApplicationPattern/issues/721.
Incase of ETIMEOUT/ECONNREFUSED , the service call is not reaching the destination and so the service record shall not be recorded to the EATL application from the server side. This is the reason for marking the status code for such cases to 404 and further we are recording the services to EATL from the client side. Kindly let me know whether this scenario shall also be take care.

Reply:
Yes, this scenario works as expected. Running RegistryOffice only, calling register-application results in:

```
POST /v1/register-application 204 587.549 ms - -
Request errored with Error: connect ECONNREFUSED 127.0.0.1:3002
Request errored with Error: connect ECONNREFUSED 127.0.0.1:3006
recordServiceRequestFromClient - record service request from client with body {"x-correlator":"550e8400-e29b-11d4-a716-446655440000","trace-indicator":"1.3.1.1","user":"User Name","originator":"RegistryOffice","application-name":"TypeApprovalRegister","release-number":"2.0.1","operation-name":"/v1/regard-application","response-code":408} failed with response status: 408
recordServiceRequestFromClient - record service request from client with body {"x-correlator":"550e8400-e29b-11d4-a716-446655440000","trace-indicator":"1.3.1.2","user":"User Name","originator":"RegistryOffice","application-name":"ApplicationLayerTopology","release-number":"2.0.1","operation-name":"/v1/update-ltp","response-code":408} failed with response status: 408
```